cmd/get/apps/command.go

```diff
@@ -39,7 +40,7 @@ type Config struct {
 	Logger     micrologger.Logger
 	FileSystem afero.Fs
 
-	K8sConfigAccess clientcmd.ConfigAccess
+	ConfigFlags *genericclioptions.RESTClientGetter
```

Comment:
Instead of the old k8sconfigaccess attribute we point to the global configflags in each command config.